po/hi.po

```diff
@@ -0,0 +1,117 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.zefr0x.hashes package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Scrambled777 <weblate.scrambled777@simplelogin.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: io.github.zefr0x.hashes\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-02-29 23:09+0300\n"
+"PO-Revision-Date: 2024-04-01 18:42+0530\n"
+"Last-Translator: Scrambled777 <weblate.scrambled777@simplelogin.com>\n"
+"Language-Team: Hindi\n"
+"Language: hi\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Gtranslator 46.0\n"
+
+#: data/io.github.zefr0x.hashes.desktop.in:5
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:6 hashes/ui/main_window.py:29
+msgid "Hashes"
+msgstr "Hashes"
+
+#: data/io.github.zefr0x.hashes.desktop.in:6
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:7
+msgid "Identify hashing algorithms"
+msgstr "हैशिंग कलन विधि को पहचानें"
+
+#: data/io.github.zefr0x.hashes.desktop.in:7
+msgid "Hash Identifier"
+msgstr "हैश पहचानकर्ता"
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;detect;identify;हैश;हैशकैट;जॉन;पता लगाएं;पहचाने;"
```

Comment:
Normally I include both English and translated search terms. Useful for multi-language input.

If you want then, I will just keep translated one.

Reply:
> Useful for multi-language input.

Application launchers should search in any available language. If not, than this should be reported as a bug or requested feature.

> If you want then, I will just keep translated one.

It would be better.

Reply:
Okay, editing then. Thanks.